### ISSUE TITLE
ZipRipper-Passwords.txt Not Being Created

### ISSUE THREAD

**Issue by Syeph0n** [illsk1lls/ZipRipper#11]
Pretty much as the titles says, after a successful session it doesn't create the passwords.txt file.

Taking a look at the previously opened issue about this I can tell you that `ECHO %UserProfile%`, and `CD %UserProfile%\Desktop` both yield/navigate to the correct directory. 

Additionally taking a look at the script itself I tried `ECHO %SystemDrive%\Users\%UserName%\Desktop` which also outputs correctly. 

_However as a side note_ when on my E: drive I can't seem to get CMD to navigate to the directory using `cd %SystemDrive%\Users\%UserName%\Desktop`, it doesn't return an error or anything, just doesn't do anything at all lol.

Though I've tried running ZipRipper on both C: and E: drives and it didn't create the file regardless.

I've also tried manually creating a `ZipRipper-Passwords.txt` file on my desktop. _It did not write to that either._

There also doesn't seem to be any other errors outputted to the CLI while it's running. 

Hopefully you can help!

-Edit as I forgot to mention, _I am **not** using a redirected desktop through onedrive._

**Comment by illsk1lls** [comment]
Is the script completing? Do you see a password displayed in the main window in a yellow/orange color?

**Comment by Syeph0n** [comment]
It doesn't display anything in yellow/orange. However it does say "Session completed"

And then tells me "Resume is UNAVAILABLE". 

I guess I just assumed the session being completed meant that it had found the correct password. Is it possible it's just failing before finding anything?

(I'll post a screenshot of the terminal in just a minute)

**Comment by illsk1lls** [comment]
It is crashing, resume unavailable means there is no progress to save, the files created during the process arent there…

are you using the latest version?

its possible theres an issue with the zip

can you try making a pw protected zip with 7zip with the password “password” and see if the script completes correctly?

**Comment by Syeph0n** [comment]
I should have mentioned that I was using a wordlist I downloaded and not the default wordlist.

That is what appears to have been the issue; I setup a dummy .rar file with a known password to rule out whether or not it was an issue with a corrupt archive. Had the same weird behavior on the dummy file, swapped to the default wordlist and then everything ran correctly. _(Looking back I seriously should have tried that earlier lol)_

It found the password and ZipRipper-Passwords.txt was created properly, so all seems to be good there.

After that I went and checked the wordlist I had downloaded (rockyou), and somehow it got truncated down to maybe 10 words. Literally no idea what happened other than maybe corruption during download.

Either way just for the sake of science I copied the 10 words over to a freshly created .txt file and loaded that as the word list, it crashed it in the same manner. 

So it appears a very small wordlist was causing the crashes! Mystery solved!
